ports/unix/variants/standard/mpconfigvariant.h

```diff
@@ -27,18 +27,5 @@
 // Set base feature level.
 #define MICROPY_CONFIG_ROM_LEVEL (MICROPY_CONFIG_ROM_LEVEL_EXTRA_FEATURES)
```

Comment:
Similarly, could be less confusing in the long run if this variant was called "extra"?

Reply:
Not sure... It's the standard (i.e. default) variant, it just happens to use the "extra" feature set.